compiler/src/dotty/tools/dotc/transform/PostTyper.scala

```diff
@@ -472,6 +488,9 @@ class PostTyper extends MacroTransform with InfoTransformer { thisPhase =>
               case tpe => tpe
             }
           )
+        case Assign(lhs @ Select(qual, _), _) =>
+          markVarAccess(lhs, qual)
+          super.transform(tree)
```

Comment:
Do I understand correctly that `private var`s don't get a setter then? When mutated, they are directly `Assign`ed instead of having an `apply(setter, ...)`?

Reply:
The setter gets added in a later phase, `Getters`.